020-Connect/010-Postgres/001-connect.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: Connect to Postgres
+navTitle: Connect
+keywords: ['postgres', 'connect', 'dsn']
+description: Learn how to connect to Postgres
+slug: postgres/connect
+published: true
+---
+
+<Alert status="info">The Postgres wire protocol is still considered BaseAudioContext.</Alert>
+
+This page shows you how to connect to Postgres, by explaining how a connection string is built, the different parameters and examples to connect via [psql](https://www.postgresql.org/docs/current/app-psql.html).
+
```

Comment:
probably worth adding some more intro text about Xata allowing direct access to underlying postgres, but this is a good start

Reply:
100%, I want to get the page outline first and add more meat to the bone.

Reply:
Intro text in place